apps/desktop/src/components/nav-bar.tsx

```diff
@@ -8,13 +8,13 @@ import {
   ChevronsRightLeft,
   type LucideIcon,
 } from "lucide-react";
+import { useCanary } from "@/hooks/use-platform-info";
```

Comment:
i hate to be the one for nitting but this should be `isCanary` because `use*` is a react naming convention for hooks.

https://react.dev/reference/rules/rules-of-hooks

Reply:
ya wili change